aws.env.encrypted

```diff
@@ -0,0 +1,2 @@
+cloudbees:v1
```

Comment:
I don't see any reference to this file. What is in it? We sometimes add a file `aws.env.example` to at least show what variables are needed. For example:

```env
AWS_SECRET_ACCESS_KEY=
AWS_ACCESS_KEY_ID=
AWS_REGION=
```